WooCommerce/Classes/ViewRelated/Orders/OrdersRootViewController.swift

```diff
@@ -439,8 +486,20 @@ private extension OrdersRootViewController {
     }
 }
 
+// MARK: - Helpers
+private extension OrdersRootViewController {
+    /// Validates the passed value by returning nil if it's invalid, or the passed value otherwise
+    ///
+    func validateSentinelValue(value: Int64) -> Int64? {
+        value == Constants.invalidValue ? nil : value
+    }
+}
+
 // MARK: - Constants
 private extension OrdersRootViewController {
+    enum Constants {
+        static let invalidValue: Int64 = -1
```

Comment:
I could rename this better, something like `invalidProductID` to give more context about its meaning 🤔

Reply:
Done on https://github.com/woocommerce/woocommerce-ios/pull/9736/commits/244b26d3b1881440084c0c197d69b8edb2201588